src/shared/defs.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef DEFS_HPP
+#define DEFS_HPP
+
+#define SERVER_PORT 28801
+#define MAXSDESCLEN 80
+#define MAXBRANCHLEN 16
+#define MAXTRANS 5000   // max amount of data to swallow in 1 go
+#define VERSION_GAME 230
+
+#endif
```

Comment:
For this file and also serverinfo.hpp: We use .h as file extension for all other header files. Maybe we should keep it the same (until we change all of them if we ever do that).

Reply:
Also, this change is not strictly related to the topic of the PR, as far as I can see (it also missed a few other definitions).

Can someone please create an issue so we can move all remaining definitions into this file properly, later on? Please don't forget to include a link to this comment.